frontend/plugins/vuex-orm-axios.js

```diff
@@ -70,13 +70,13 @@ export default ({ $axios, app }) => {
         break;
       case 404:
         Notification.dispatch("notify", {
-          message: "Warning: " + error.response.data.detail,
+          message: "Warning: " + messageDetail.params.detail,
```

Comment:
Some code smeller recommends to concatenate strings with backtick like so : `Warning : ${messageDetail.params.detail}`